include/swift/AST/Decl.h

```diff
@@ -8591,6 +8591,8 @@ class MacroDecl : public GenericContext, public ValueDecl {
 /// declaration/expression nodes.
 struct MacroExpansionInfo : ASTAllocated<MacroExpansionInfo> {
   SourceLoc SigilLoc;
+  DeclNameRef ModuleName;
```

Comment:
There's so syntactic change to macro expansion exprs/decls, but these two fields were added so that attached macros can continue to use the same code path in the constraint system to resolve macro refs.